src/figuresClasses.ts

```diff
@@ -1,19 +1,71 @@
 export interface Figure {
-
+  shape: string,
+  color: 'red' | 'green' | 'blue',
```

Comment:
It's better to create types for defining values that can be in shape and color properties